jenkins_pipelines/environments/common/pipeline.groovy

```diff
@@ -30,16 +30,16 @@ def run(params) {
             stage('Sanity Check') {
                 sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:sanity_check'"
             }
-            
+
             stage('Core - Setup') {
                 sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:core'"
-                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:reposync'"
+                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS_ENABLED=${params.run_long_tests}; cd /root/spacewalk/testsuite; rake cucumber:reposync'"
             }
             stage('Core - Initialize clients') {
                 sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:init_clients'"
             }
             stage('Secondary features') {
-                def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
+                def statusCode1 = sh script:"./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'export LONG_TESTS_ENABLED=${params.run_long_tests}; cd /root/spacewalk/testsuite; rake cucumber:secondary'", returnStatus:true
```

Comment:
I'm not sure neither. Just play in your console, see if the value stay or it's only there during the command.

Reply:
in the Uyuni PR introducing this feature, the env variable is `LONG_TESTS`, the export needs to be matching it

Reply:
Renamed to `long_tests` to match, I will try locally to see if the secondary is needed.

Reply:
I don't think the env variables are persistent. From what I see in the current run of uyuni at least. But I will check again when we merge this.